development/getting_code_on_the_robot.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: Getting Code on the Robot
+---
+
+Getting Code on the Robot
+======================
+
+Getting your code on to the robot is quite simple.
+You will need to put your code on a USB drive. Which will need to be formatted with either FAT32 or exFAT.
+
+<details>
+<summary>Windows</summary>
+1. Open your code in File Explorer
+2. Select all of your code files (<kbd>Control + A</kbd> to select all files)
+3. Right-click the files
+4. Click Compress to Zip file
+5. Name the newly created file `robot`
```

Comment:
```suggestion
5. Name the newly created file `robot` (`robot.zip`)
```

Reply:
Hrm, the gotcha here is around Windows' default behaviour of hiding file extensions. Not sure what to do about that.